Code/GraphMol/Chirality.cpp

```diff
@@ -1124,9 +1133,22 @@ void findPotentialStereoBonds(ROMol &mol, bool cleanIt) {
             if (begAtomNeighbors.size() > 0 && endAtomNeighbors.size() > 0) {
               if ((begAtomNeighbors.size() == 2) &&
                   (endAtomNeighbors.size() == 2)) {
-                // if both of the atoms have 2 neighbors (other than the one
-                // connected
-                // by the double bond) and ....
+// if both of the atoms have 2 neighbors (other than the one
+// connected
+// by the double bond) and ....
+#if 0
```

Comment:
Not normally, and I almost removed it. But I'm about 95% sure that this code is going to be revisited relatively soon to try and solve the problem that it doesn't really work the way that I would like it to with dependent stereochemistry. The current status is better than it was before, but it's not done yet. And when I revisit it, I don't see any reason to have to rewrite this.